src/general/storageclass/src.rego

```diff
@@ -0,0 +1,48 @@
+package k8sstorageclass
+
+is_pvc(obj) {
+  obj.apiVersion == "v1"
+  obj.kind == "PersistentVolumeClaim"
+}
+
+is_statefulset(obj) {
+  obj.apiVersion == "apps/v1"
+  obj.kind == "StatefulSet"
+}
+
+violation[{"msg": msg}] {
+  count(data.inventory.cluster["storage.k8s.io/v1"]["StorageClass"]) == 0
+  msg := sprintf("storageclasses not synced. Gatekeeper is misconfigured.", [])
```

Comment:
This could also be due in lag waiting for objects to sync, so worth mentioning that to avoid sending users down a bad path.

Are there any cases when StorageClass would be expected to not have any members?

Reply:
/me shrugs

Its possible, but StorageClasses are highly likely to be static. so the vast majority of the time, no StorageClasses are seen, gateeper needs to be reconfigured.  But... maybe we lighten it to "may be misconfigured"?